webmention.php

```diff
@@ -33,6 +33,11 @@ class Webmention_Plugin {
 	 * Initialize Webmention Plugin
 	 */
 	public static function init() {
+		// Add a new feature type to posts for webmentions
+		add_post_type_support( 'post', 'webmentions' );
+		//	if ( 1 == get_option( 'webmention_support_pages' ) ) {
```

Comment:
That is my fault. I forgot to uncomment before I commented. I was debugging and forgot to remove the comment I put in while I was testing the feature.